---
fixes:
  - |
    Pin urllib3 to <2.0.0 for the JJB cleanup

    The latest version of module breaks compatibility with python-jenkins.

    Error:

    .. code-block:: bash

       ValueError: Timeout value connect was <object object at
       0x7fe57a4948a0>, but it must be an int, float or None.

    Reference:

       `Launchpad#2018567 <https://bugs.launchpad.net/python-jenkins/+bug/2018567>`
